packages/_private/e2e-demo-test/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@synergy-design-system/vanilla-demo",
+  "private": true,
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "start": "npm run dev",
+    "dev": "vite",
+    "test": "run-p vanilla_test vue_test react_test ng_test",
```

If we apply the naming changes, this would be something more like

```suggestion
    "test": "run-p vanilla.run vue.run react.run ng.run",
```